test/format/ops.c

```diff
@@ -1046,6 +1046,7 @@ ops(void *arg)
                 intxn = false;
             }
 
+            /* TODO: Enable prefetch here? */
```

Comment:
@sueloverso, do you know if we should enable prefetch here?

Reply:
I think it would make sense to do it, as this could result in `op = READ` which could result in a read operation where prefetch could be useful. What do you think, @kommiharibabu?

Reply:
Prefetch can be useful whenever the cursor is performing next/prev. I do not think it is useful here.
How about enabling prefetch when we are doing mirror validation?

Reply:
Right, I will remove the comment then.
For the mirror validation, see what I have done [here](https://github.com/wiredtiger/wiredtiger/pull/10590/files#diff-974b14a79fdddc7875af9b71cbb340e0a6e1324504f0b2d547b74eb63f9a7f8b). Prefetch would be enabled 20% of the time for any kind of validation.

Reply:
Removed in d3954c0a7a72b352e737bb2e3d5e35ef3b2cfb11.